fleet/error.go

```diff
@@ -0,0 +1,29 @@
+package fleet
+
+import (
+	"github.com/juju/errgo"
+)
+
+var (
+	maskAny = errgo.MaskFunc(errgo.Any)
+)
+
+var ipNotFoundError = errgo.New("ip not found")
```

Comment:
so far we have made errors public - any special reason for making this now private?